docs/release.md

```diff
@@ -71,28 +71,42 @@ If you want to push changes to the `release-X.Y` release branch, you have to che
     ```bash
     # Get version and remove the leading 'v'
     VERSION=$(cat VERSION | sed "s/^v//")
+
+    # Change the version and appVersion in Chart.yaml
+    # On Linux
     sed -i "s/^version.*/version: ${VERSION}/" charts/spark-operator-chart/Chart.yaml
     sed -i "s/^appVersion.*/appVersion: ${VERSION}/" charts/spark-operator-chart/Chart.yaml
+
+    # On MacOS
+    sed -i '' "s/^version.*/version: ${VERSION}/" charts/spark-operator-chart/Chart.yaml
+    sed -i '' "s/^appVersion.*/appVersion: ${VERSION}/" charts/spark-operator-chart/Chart.yaml
+    ```
+
+3. Update the Helm chart README:
+
+    ```bash
+    make helm-docs
     ```
 
-3. Commit the changes:
+4. Commit the changes:
 
     ```bash
     git add VERSION
     git add charts/spark-operator-chart/Chart.yaml
-    git commit -s -m "Release $VERSION"
-    git push
+    git add charts/spark-operator-chart/README.md
+    git commit -s -m "Spark Operator Official Release v${VERSION}"
+    git push origin release-X.Y
```

Comment:
Do we need this, since we make the commit to the release branch via `release` GitHub action ?

Reply:
I think this is needed, for that the `release` GitHub action is used to create a new draft release, not to make a commit to the release branch. So we need to update chart `version `/`appVersion` and README at the same time when we modify the `VERSION` file.